local-volume/provisioner/pkg/common/common.go

```diff
@@ -49,6 +51,8 @@ const (
 
 	// EventVolumeFailedDelete copied from k8s.io/kubernetes/pkg/controller/volume/events
 	EventVolumeFailedDelete = "VolumeFailedDelete"
+	// Location inside the provisioner container where configMap volume is mounted
+	provisionerConfigPath = "/etc/provisioner/config/..data/"
```

Comment:
This path looks odd.

Reply:
That is how configMap gets mounted when used as a volume.  I am not sure where it came from but it is how it is now. I do not think I can do much about it.